# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rezound/float.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

do not throw because of floating point representation 

--- rezound-0.12.2beta/src/backend/settings.cpp_orig	2006-03-06 11:49:50.130992750 +0100
+++ rezound-0.12.2beta/src/backend/settings.cpp	2006-03-06 15:42:48.244570250 +0100
@@ -222,7 +222,7 @@
 	GET_SETTING("DesiredOutputBufferCount",gDesiredOutputBufferCount,int)
 		gDesiredOutputBufferCount=max(2,gDesiredOutputBufferCount);
 	GET_SETTING("DesiredOutputBufferSize",gDesiredOutputBufferSize,unsigned)
-		if(gDesiredOutputBufferSize<256 || log((double)gDesiredOutputBufferSize)/log(2.0)!=floor(log((double)gDesiredOutputBufferSize)/log(2.0)))
+		if(gDesiredOutputBufferSize<256 || (gDesiredOutputBufferSize & gDesiredOutputBufferSize-1))
 			throw runtime_error(string(__func__)+" -- DesiredOutputBufferSize in "+gSettingsRegistry->getFilename()+" must be a power of 2 and >= than 256");
 
 
